build-system/compile/log-messages.js

```diff
@@ -33,23 +33,33 @@ const formats = {
   [`${pathPrefix}.simple.json`]: ({message}) => message,
 };
 
-/** @return {!Promise<!Array<!Object>>} */
-const extractedItems = () => fs.readJson(extractedPath).then(Object.values);
+/** @return {!Promise<?Array<!Object>>} */
+async function extractedItems() {
+  try {
+    return Object.values(await fs.readJson(extractedPath));
+  } catch (_) {
+    return null;
+  }
+}
 
 /**
  * Format extracted messages table in multiple outputs, keyed by id.
  * @return {!Promise}
  */
 async function formatExtractedMessages() {
   const items = await extractedItems();
-  return Promise.all(
+  if (!items) {
+    log(yellow('Skipped formatting log message table'));
```

Comment:
Does this message get printed for every build step? If so, I suggest dropping it to make the logs less chatty.

Reply:
Done.